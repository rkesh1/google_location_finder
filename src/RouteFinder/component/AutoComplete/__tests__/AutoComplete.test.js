import React from 'react';
import { shallow } from 'enzyme';
import Autocomplete from '../index';


describe('<Autocomplete /> Component Test Case ', () => {
  let wrapper;
  const props = {
    additionClass: '',
    placeholder: '',
    type: 'text',
    id: 'xyz',
    name: 'pqr',
    labelTitle: 'Title',
  };
  /**
   * Renders a shallow version of the Autocomplete
   */
  function renderShallow() {
    wrapper = shallow(<Autocomplete {...props} />);
  }


  it('Autocomplete Component render', () => {
    renderShallow();
    expect(wrapper).toMatchSnapshot();
  });

  it('Autocomplete have label tag', () => {
    renderShallow();
    expect(wrapper.find('.input-label').length).toBe(1);
  });

  it('Autocomplete have input tag', () => {
    renderShallow();
    expect(wrapper.find('input').length).toBe(1);
  });

});
