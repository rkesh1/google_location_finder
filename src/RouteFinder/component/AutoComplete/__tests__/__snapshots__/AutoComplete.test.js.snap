// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Autocomplete /> Component Test Case  Autocomplete Component render 1`] = `ShallowWrapper {}`;
